### PR TITLE
Feature/2097 manage evaluations round wise segments

### PR DESCRIPTION
**Pull request by LAKSHYA0501**
Targets #2097 
### Description
1. Made changes in the UI from tables to accordions.
2. Added Round wise evaluations in accrodions and a drop down of respective segments.
3. fixed delete button.

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/HR/Http/Controllers/EvaluationController.php [hunk 1]
```diff
@@ -20,12 +20,24 @@ class EvaluationController extends Controller
      */
     public function index(Request $request)
     {
+        $segmentedRounds = [];
         $segments = Segment::all();
         $rounds = Round::select('id', 'name')->get();
+        foreach ($rounds as $value) {
+            $attr = Round::find($value->id)->evaluationSegments;
+            $segmentedRounds[$value->id] = $attr;
```
Comment (P4NK4J) [inline Modules/HR/Http/Controllers/EvaluationController.php:28]: what is $segmentedRounds ? It doesn't make sense to me, can the variable be like roundWithSegments if that's what you meant ?

File: Modules/HR/Resources/views/evaluation/index.blade.php [hunk 2]
```diff
@@ -25,33 +24,71 @@
...
-                    <i v-on:click="editSegment({{ $segment }})" class="fa fa-edit fz-20 text-theme-green"></i>
-
-                    <i v-on:click="removeSegment({{ $segment }})" class="fa fa-trash fz-20 text-theme-red"></i>
-                </td>
-            </tr>
+        <div class="container">
+             @foreach ($rounds as $round ) 
+            <?php
+                $count = 0;
+
```
Comment (P4NK4J) [inline Modules/HR/Resources/views/evaluation/index.blade.php:31]: remove this space

File: Modules/HR/Resources/views/evaluation/index.blade.php [hunk 3]
```diff
@@ -25,33 +24,71 @@
...
+             @foreach ($rounds as $round ) 
+            <?php
+                $count = 0;
+                
+                ?>      
+            @foreach ($attr as $key=>$segment)
+            @if ($round->id == $key)
+            <div class="accordion" id="accordion">
+                <div class="accordion-header">
+                    <div class="card" style="background-color:#E7E6E0">
```
Comment (P4NK4J) [inline Modules/HR/Resources/views/evaluation/index.blade.php:37]: Let's use bootstrap class for this, maybe something like `bg-color-variable` defined in variables.scss

File: Modules/HR/Resources/views/evaluation/index.blade.php [hunk 4]
```diff
@@ -25,33 +24,71 @@
...
+                            <div class="form-group col-md p-3">
+                                <div class="icon-arrow-down position-relative ml-3 c-pointer" data-toggle="collapse" data-target="#segmentCollapse_{{ $round->id }}">
+                                    <i class="fa fa-arrow-down"></i>
+                                </div>
+                            </div>    
+                        </div>
+                    </div> 
+                </div>
+                <div id="segmentCollapse_{{ $round->id }}" class="collapse <?php if($count==0){
+                    echo "show";}?>" aria-labelledby="headingOne">
```
Comment (P4NK4J) [inline Modules/HR/Resources/views/evaluation/index.blade.php:55]: Let's use ternary operators here `$count == 0 ? 'show' : ' '`

File: Modules/HR/Resources/views/evaluation/index.blade.php [hunk 5]
```diff
@@ -25,33 +24,71 @@
...
+                                <td>
+                                    <h5>{{ $value->parameters->sum('marks') }}</h5>
+                                </td>
+                                
+                                <td>
+                                    <i v-on:click="editSegment({{ $value }})" class="fa fa-edit fz-20 text-theme-green"></i>
+                                
+                                    <i v-on:click="removeSegment({{ $value }})" class="fa fa-trash fz-20 text-theme-red"></i>
+                                </td>
+                            </tr>
```
Comment (P4NK4J) [inline Modules/HR/Resources/views/evaluation/index.blade.php:80]: Can you explain what's inside $round, $attr and $segment ? I don't think three nested for loops is a good practice

**Review by P4NK4J — CHANGES_REQUESTED** [review]
Left some comments, please check

File: Modules/HR/Resources/views/evaluation/index.blade.php (hunk 4 above)
Comment (LAKSHYA0501) [reply Modules/HR/Resources/views/evaluation/index.blade.php:55]: ok done for this?

File: Modules/HR/Resources/views/evaluation/index.blade.php (hunk 2 above)
Comment (LAKSHYA0501) [reply Modules/HR/Resources/views/evaluation/index.blade.php:31]: done

File: Modules/HR/Http/Controllers/EvaluationController.php (hunk 1 above)
Comment (LAKSHYA0501) [reply Modules/HR/Http/Controllers/EvaluationController.php:28]: so i was not sure with name that's why i used this one, no worries i changed it.

File: Modules/HR/Resources/views/evaluation/index.blade.php (hunk 3 above)
Comment (LAKSHYA0501) [reply Modules/HR/Resources/views/evaluation/index.blade.php:37]: done

File: Modules/HR/Resources/views/evaluation/index.blade.php (hunk 5 above)
Comment (LAKSHYA0501) [reply Modules/HR/Resources/views/evaluation/index.blade.php:80]: In $round we have id and name of all rounds
in $attr we have round_id wise data of all segments in arrays
in  $segment we have different parameters or data of segments 

So should i change them?

**Review by P4NK4J — APPROVED** [review]
Looks good.

**Pull request merged**